src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/TabularOverloadsBlanksAndErrors.txt

```diff
@@ -252,10 +251,11 @@ Table({Value:0},{Value:0},{Value:0})
 Table()
 
 >> Concatenate(Blank(), Filter([1], Value > 10))
-Blank()
+Table()
 
 >> Concatenate(Blank(), "test")
 "test"
 
+// suspect, may be different on Canvas requiring an override
```

Comment:
There are three tests that give different results in Canvas.  We need to validate that these are the right answers and that Canvas should snap to these semantics with Power Fx 1.0.  @CarlosFigueiraMSFT

Reply:
Checking why this doesn't work properly in canvas.

Reply:
C# has the correct behavior.  Carlos is looking into why Canvas is different.